docs/settings.rst

```diff
@@ -288,7 +288,14 @@ Basic settings
 
    A list of tags for Typogrify to ignore. By default Typogrify will ignore
    ``pre`` and ``code`` tags. This requires that Typogrify version 2.0.4 or
-   later is installed
+   later is installed.
+
+.. data:: TYPOGRIFY_OMIT_FILTERS = []
+
+   A list of Typogrify filters to skip. Allowed values are: ``'amp'``,
+   ``'smartypants'``, ``'caps'``, ``'initial_quotes'``, ``'widont'``. By
+   default, no filter is omitted (in other words, all filters get applied). This
+   setting requires that Typogrify version 2.0.8 or later is installed.
```

Comment:
Given the additional to-be-released functionality, in keeping with [semantic versioning](https://semver.org/) I expect the next release to be `2.1.0`.

Reply:
Got it. I'll make the change accordingly.